<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Rem</class>
 <widget class="QMainWindow" name="Rem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rem</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QToolButton" name="Node2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>200</y>
      <width>23</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QToolButton" name="Node8">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>200</y>
      <width>23</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QToolButton" name="Node11">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>11</string>
    </property>
   </widget>
   <widget class="QToolButton" name="Node3">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>200</y>
      <width>23</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QToolButton" name="Node10">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QToolButton" name="Node6">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>200</y>
      <width>23</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QToolButton" name="Node5">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>200</y>
      <width>23</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QToolButton" name="Node9">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>260</y>
      <width>23</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QToolButton" name="Node7">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>260</y>
      <width>23</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QToolButton" name="Node4">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>260</y>
      <width>23</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QToolButton" name="Node1">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>200</y>
      <width>23</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QToolButton" name="Node12">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>12</string>
    </property>
   </widget>
   <widget class="QWidget" name="sideWidget" native="true">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>330</y>
      <width>341</width>
      <height>131</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>Sensor with no user detected</string>
     </property>
     <property name="background">
      <color>
       <red>0</red>
       <green>255</green>
       <blue>0</blue>
      </color>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Sensor with Potential User detected</string>
     </property>
     <property name="background">
      <color>
       <red>255</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Registered Secondary User Without Grant </string>
     </property>
     <property name="background">
      <color>
       <red>255</red>
       <green>255</green>
       <blue>0</blue>
      </color>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Registered User With Accepted Grant and Transmitting</string>
     </property>
     <property name="background">
      <color>
       <red>0</red>
       <green>0</green>
       <blue>255</blue>
      </color>
     </property>
    </row>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>759</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
